kxml_compiler/creator.cpp

```diff
@@ -237,7 +237,8 @@ ClassDescription Creator::createClassDescription(const Schema::Element &element)
 {
     ClassDescription description(Namer::getClassName(element.name()));
 
-    foreach (Schema::Relation r, element.attributeRelations()) {
+    auto attributeRelations = element.attributeRelations();
```

Comment:
```suggestion
    const auto attributeRelations = element.attributeRelations();
```

Otherwise the range-for on the next line will detach (deep-copy the list).

(repeats)